source/store/help_button_text.py

```diff
@@ -2,23 +2,30 @@
                "the numeric invariants and operations. Each one must be applied to a graph, as a function “f(G)” and" \
                " compositions. Conditions can be combined using logical connectors AND or OR. <br> Examples:" \
                " <br>\u03c9(G) == a(\u2113(G)) <br> \u03bb \u2081(G) > (\u03bb\u2081(c(G)))/2 <br>" \
-           "\u03c9(G) == a(\u2113(G)) OR \u03bb \u2081(G) > (\u03bb\u2081(c(G)))/2 </p>"
+               "\u03c9(G) == a(\u2113(G)) OR \u03bb \u2081(G) > (\u03bb\u2081(c(G)))/2 <br> <br> You can access our " \
+               "<a href=""https://github.com/GraphFilter/GraphFilter/wiki/Dictionary""> dictionary</a> " \
+               "to find out more about each invariant.</p>"
 
 method = "<p> The 'Filter graphs'' will filter a list of graphs and return all those that satisfy the conditions " \
-             "marked in the previous screens. <br> The 'Find Counterexample' will run until it finds a graph that " \
-             "does not meet the conditions, if any. </p>"
+             "marked in the previous screens. <br> The 'Find an example' will run until it finds a first graph that " \
```

Comment:
Filter Graphs: The conditions will be marked on the following screens (not previously)

Reply:
Done!